tools/proto-compiler/src/constants.rs

```diff
@@ -2,11 +2,30 @@
 
 /// Tendermint repository URL.
 pub const TENDERMINT_REPO: &str = "https://github.com/tendermint/tendermint";
-// Commitish formats:
-// Tag: v0.34.0-rc4
-// Branch: master
-// Commit ID (full length): d7d0ffea13c60c98b812d243ba5a2c375f341c15
-pub const TENDERMINT_COMMITISH: &str = "v0.34.21";
+
+/// Information on a Tendermint snapshot to generate prost structures from.
+pub struct TendermintVersion {
+    /// Identifier to use in module names.
+    pub ident: &'static str,
+    /// A commitish reference in the tendermint git repository, for example:
+    ///
+    /// - Tag: `v0.34.0-rc4`
+    /// - Branch: `main`
+    /// - Commit ID (full length): `d7d0ffea13c60c98b812d243ba5a2c375f341c15`
+    pub commitish: &'static str,
+}
+
+/// All Tendermint versions to generate code for
+pub const TENDERMINT_VERSIONS: &[TendermintVersion] = &[
+    TendermintVersion {
+        ident: "v0_34",
+        commitish: "v0.34.24",
+    },
+    TendermintVersion {
+        ident: "v0_37",
+        commitish: "v0.37.0-alpha.1",
```

Comment:
This is the last tag in Tendermint Core where the proto files had bundled Google well known types, before switching to buf to refer to dependencies. Adapting to buf will require more changes to the proto-compiler tool, and I have already done this already once in an older branch.

Here I'm leaving the migration to buf, and switching to CometBFT, as follow-on tasks.
There is no difference in protobuf message formats with the latest state in CometBFT branch `v0.37.x` (as per commit d26bdfe1d424ff24c9676fee5bef9a180870315e).